libs/plpgsql.json

```diff
@@ -0,0 +1,46 @@
+  {"id": "plpgsql.pg_buffercache", "imports": ["pg_buffercache"], "name": "pg_buffercache", "repo": "postgres/postgres/tree/master/contrib/pg_buffercache", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pgcrypto", "imports": ["pgcrypto"], "name": "pgcrypto", "repo": "postgres/postgres/tree/master/contrib/pgcrypto", "tags": ["security"], "tech": ["cryptography"]},
+  {"id": "plpgsql.pg-freespacemap", "imports": ["pg_freespacemap"], "name": "pg_freespacemap", "repo": "postgres/postgres/tree/master/contrib/pg_freespacemap", "tags": ["free-space-map"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pg-prewarm", "imports": ["pg_prewarm"], "name": "pg_prewarm", "repo": "postgres/postgres/tree/master/contrib/pg_prewarm", "tags": ["caching"], "tech": ["faster-code"]},
+  {"id": "plpgsql.pgrowlocks", "imports": ["pgrowlocks"], "name": "pgrowlocks", "repo": "postgres/postgres/tree/master/contrib/pgrowlocks", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pg-stat-statements", "imports": ["pg_stat_statements"], "name": "pg_stat_statements", "repo": "postgres/postgres/tree/master/contrib/pg_stat_statements", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pgstattuple", "imports": ["pgstattuple"], "name": "pgstattuple", "repo": "postgres/postgres/tree/master/contrib/pgstattuple", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.pg-trgm", "imports": ["pg_trgm"], "name": "pg_trgm", "repo": "postgres/postgres/tree/master/contrib/pg_trgm", "tags": ["string-search"], "tech": ["faster-code"]},
+  {"id": "plpgsql.pg-visibility", "imports": ["pg_visibility"], "name": "pg_visibility", "repo": "postgres/postgres/tree/master/contrib/pg_visibility", "tags": ["db-analysis"], "tech": ["eng-tools"]},
+  {"id": "plpgsql.postgres-fdw", "imports": ["postgres_fdw"], "name": "postgres_fdw", "repo": "postgres/postgres/tree/master/contrib/postgres_fdw", "tags": ["db-link"], "tech": ["app-arch"]},
+  {"id": "plpgsql.seg", "imports": ["seg"], "name": "seg", "repo": "postgres/postgres/tree/master/contrib/seg", "tags": ["intervals"], "tech": ["data-structures"]},
+  {"id": "plpgsql.sepgsql", "imports": ["sepgsql"], "name": "sepgsql", "repo": "postgres/postgres/tree/master/contrib/sepgsql", "tags": ["access-control"], "tech": ["security"]},
+  {"id": "plpgsql.spi", "imports": ["spi"], "name": "spi", "repo": "postgres/postgres/tree/master/contrib/spi", "tags": ["spi"], "tech": ["api"]},
+  {"id": "plpgsql.sslinfo", "imports": ["sslinfo"], "name": "sslinfo", "repo": "postgres/postgres/tree/master/contrib/sslinfo", "tags": ["ssl"], "tech": ["security"]},
+  {"id": "plpgsql.tablefunc", "imports": ["tablefunc"], "name": "tablefunc", "repo": "postgres/postgres/tree/master/contrib/tablefunc", "tags": ["plpgsql-c"], "tech": ["interoperability"]},
+  {"id": "plpgsql.tcn", "imports": ["tcn"], "name": "tcn", "repo": "postgres/postgres/tree/master/contrib/tcn", "tags": ["triggers"], "tech": ["utils"]},
```

Comment:
tech == app-arch. if are thinking about triggers/notifications, you are thinking about architecture.